app/api/talk.coffee

```diff
@@ -12,7 +12,18 @@ talkClient = new JSONAPIClient 'https://talk-staging.zooniverse.org',
   'Content-Type': 'application/json'
   'Accept': 'application/json'
 
-talkClient.headers = apiClient.headers
+authClient.listen 'change', ->
```

Comment:
I added this back in because I was having headers issues I can't recall right now, but will have a better auth solution for talk in down the line.